content/en/docs/ops/configuration/traffic-management/tls-configuration/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Understanding TLS Configuration
+linktitle: TLS Configuration
+description: How to configure TLS settings to secure network traffic.
+weight: 30
+keywords: [traffic-management,proxy]
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+One of Istio's most important features is the ability to lock down and secure network traffic to, from,
+and within the mesh. However, configuring TLS settings can be confusing and a common source of misconfiguration.
+This document attempts to explain the various connections involved when sending requests in Istio and how
```

Comment:
Do we send requests in Istio or maybe it's through Istio?

Reply:
or maybe "with Istio"? I think it's OK as is.